roles/matrix-synapse/templates/synapse/homeserver.yaml.j2

```diff
@@ -914,15 +914,15 @@ password_config:
 {% if matrix_synapse_email_enabled %}
 email:
    enable_notifs: true
-   smtp_host: {{ matrix_synapse_email_smtp_host|to_json }}
-   smtp_port: {{ matrix_synapse_email_smtp_port|to_json }}
+   smtp_host: {{ matrix_synapse_email_smtp_host|string|to_json }}
+   smtp_port: {{ matrix_synapse_email_smtp_port|string|to_json }}
```

Comment:
`matrix_synapse_email_smtp_port` is a number (`587` by default in the role, and overriden to `8025` in `group_vars/matrix-servers`).

I'm not sure if Synapse casts it automatically (it probably does), but it's most likely better if we just leave it without `|string`. I don't see much reason for obfuscating a port number by encrypting the variable.

Using `|int` yields the correct numeric value, but I'm not sure if it's a working workaround for your encryption problem or if that requires passing it through `|string`. Or maybe `|string|int`?

I guess we better remove the `|string` part completely though.